test/jdk/java/lang/invoke/VarHandles/X-VarHandleTestMethodType.java.template

```diff
@@ -26,12 +26,19 @@
 /*
  * @test
  * @bug 8156486
+#if[Point]
```

Comment:
This change intends to make `@compile....` to be conditional.   So it can simply do:

```suggestion
#if[Point]
     * @compile -XDenablePrimitiveClasses Point.java Value.java VarHandleTestMethodType$Type$.java
#end[Point]
```

No need to duplicate all `@run` statements.

Reply:
Check